internal/daemonset/daemonset.go

```diff
@@ -187,8 +189,28 @@ func CreateDaemonSet(daemonSetName, namespace, containerName, imageWithVersion s
 	return daemonsetPods, nil
 }
 
+// Build image with version based on environment variables if provided, else use a default value
+func buildImageWithVersion() string {
+	var tnfPartnerRepo string
+	var supportImage string
+
+	if os.Getenv("TNF_PARTNER_REPO") != "" {
+		tnfPartnerRepo = os.Getenv("TNF_PARTNER_REPO")
+	} else {
+		tnfPartnerRepo = tnfPartnerRepoDef
+	}
+	if os.Getenv("SUPPORT_IMAGE") != "" {
+		supportImage = os.Getenv("SUPPORT_IMAGE")
+	} else {
+		supportImage = supportImageDef
+	}
```

Comment:
I'd write less strings:
```
tnfPartnerRepo := os.Getenv("TNF_PARTNER_REPO")
if tnfPartnerRepo == "" {
  tnfPartnerRepo = tnfPartnerRepoDef
}
supportImage := os.Getenv("SUPPORT_IMAGE")
if supportImage == "" {
  supportImage = supportImageDef
}
```

Reply:
I tried in previous commits but it failed on lint check, I'll try it again to see if it works, also checking that is not breaking our jobs

Reply:
Included, and seems that lint is working, thanks!